src/styles/components/UpcomingEvent.module.css

```diff
@@ -26,7 +26,7 @@
   content: '';
   width: 22px;
   height: 22px;
-  background-image: url('/assets/design-vectors/pointer.svg');
+  /*  background-image: url('/assets/design-vectors/pointer.svg'); */
   position: absolute;
   right: 0;
   bottom: 0;
```

Comment:
i cant make a comment, but can you remove the css on line 48 (the one that overrides the p style), it shouldnt be different than what the global styles are